images/rootfs-mini.yml.in.patch

```diff
@@ -6,9 +6,9 @@
 +  image: NEW_KERNEL_TAG
    cmdline: "rootdelay=3"
  init:
--  - linuxkit/init:a68f9fa0c1d9dbfc9c23663749a0b7ac510cbe1c
--  - linuxkit/runc:f79954950022fea76b8b6f10de58cb48e4fb3878
--  - linuxkit/containerd:1ae8f054e9fe792d1dbdb9a65f1b5e14491cb106
+-  - linuxkit/init:78fb57c7da07c4e43c3a37b27755581da087a3b6
+-  - linuxkit/runc:bf1e0c61fb4678d6428d0aabbd80db5ea24e4d4d
+-  - linuxkit/containerd:cc02c2af9c928c2faeccbe4edc78bd297ad91866
```

Comment:
Ok, so these are not only bumping the GO version, but also the version of the components themselves, right? If so -- this is again a pretty high risk to pull into a patch release.